```
error[E0599]: no variant or associated item named `Window` found for enum `CompositeTarget` in the current scope
   --> ports/servoshell/egl/ohos/simpleservo.rs:117:26
    |
117 |         CompositeTarget::Window,
    |                          ^^^^^^ variant or associated item not found in `CompositeTarget`
```

--- a/components/compositing/compositor.rs
+++ b/components/compositing/compositor.rs
@@ -2247,7 +2247,7 @@ impl IOCompositor {
     }
 
     /// Return the OpenGL framebuffer name of the most-recently-completed frame when compositing to
-    /// [`CompositeTarget::Fbo`], or None otherwise.
+    /// [`CompositeTarget::OffscreenFbo`], or None otherwise.
     pub fn offscreen_framebuffer_id(&self) -> Option<gleam::gl::GLuint> {
         self.prev_offscreen_framebuffer
             .as_ref()

--- a/ports/servoshell/egl/android/simpleservo.rs
+++ b/ports/servoshell/egl/android/simpleservo.rs
@@ -113,7 +113,7 @@ pub fn init(
         embedder_callbacks,
         window_callbacks.clone(),
         None,
-        CompositeTarget::Window,
+        CompositeTarget::ContextFbo,
     );
 
     SERVO.with(|s| {

--- a/ports/servoshell/egl/ohos/simpleservo.rs
+++ b/ports/servoshell/egl/ohos/simpleservo.rs
@@ -114,7 +114,7 @@ pub fn init(
         embedder_callbacks,
         window_callbacks.clone(),
         None, /* user_agent */
-        CompositeTarget::Window,
+        CompositeTarget::ContextFbo,
     );
 
     let servo_glue = ServoGlue::new(

